### PR TITLE
Admin client internal types

### PR DESCRIPTION
**Pull request by vytautas-karpavicius**
**What changed?**
Switch admin client to internal types. Changed immediate client usage by passing already constructed types or using thrift mappers at the call boundary. Upcoming PRs will propagate those conversions further up, until internal types reaches handlers eventually.

**Why?**
To propagate internal types throughout Cadence codebase.

**How did you test it?**
Existing tests.

**Potential risks**

### CODE REVIEW

File: client/admin/thriftClient.go [hunk 1]
```diff
@@ -23,12 +23,12 @@ package admin
 import (
 	"context"
 
+	"github.com/uber/cadence/common/types/mapper/thrift"
```
Comment (yycptt) [inline client/admin/thriftClient.go:26]: nit: please group with other cadence imports

File: service/history/replication/dlq_handler_test.go [hunk 2]
```diff
@@ -24,15 +24,18 @@ import (
 	"context"
 	"testing"
 
+	"github.com/uber/cadence/common/types"
+	"github.com/uber/cadence/common/types/mapper/thrift"
```
Comment (yycptt) [inline service/history/replication/dlq_handler_test.go:28]: same here.

File: service/history/replication/task_fetcher_test.go [hunk 3]
```diff
@@ -23,12 +23,14 @@ package replication
 import (
 	"testing"
 
+	"github.com/uber/cadence/common/types/mapper/thrift"
```
Comment (yycptt) [inline service/history/replication/task_fetcher_test.go:26]: same here

File: service/history/execution/context.go [hunk 4]
```diff
@@ -1189,10 +1191,10 @@ func (c *contextImpl) ReapplyEvents(
 	}
 	return sourceCluster.ReapplyEvents(
```
Comment (yycptt) [inline service/history/execution/context.go:1192]: Do we need `thrift.FromError()` here?

File: client/admin/thriftClient.go (hunk 1 above)
Comment (vytautas-karpavicius) [reply client/admin/thriftClient.go]: Done

File: service/history/replication/dlq_handler_test.go (hunk 2 above)
Comment (vytautas-karpavicius) [reply service/history/replication/dlq_handler_test.go]: Done

File: service/history/replication/task_fetcher_test.go (hunk 3 above)
Comment (vytautas-karpavicius) [reply service/history/replication/task_fetcher_test.go]: Done

File: service/history/execution/context.go (hunk 4 above)
Comment (vytautas-karpavicius) [reply service/history/execution/context.go]: Yes we do, thanks for noticing, fixed.

**Review by yycptt — APPROVED** [review]

**Pull request merged**